kpops/utils/pydantic.py

```diff
@@ -111,20 +113,55 @@ def json_schema_extra(schema: dict[str, Any], model: type[BaseModel]) -> None:
 class YamlConfigSettingsSource(PydanticBaseSettingsSource):
     """Loads variables from a YAML file at the project's root."""
 
-    path_to_config = Path("config.yaml")
+    log = logging.getLogger()
+
+    path_to_settings = Path()
```

Comment:
I think we should stick with `config` and not `setting`
```suggestion
    path_to_config = Path()
```